src/set_theory/cofinality.lean

```diff
@@ -495,6 +495,62 @@ theorem succ_is_regular {c : cardinal.{u}} (h : omega ≤ c) : is_regular (succ
     apply typein_lt_type }
 end⟩
 
+/--
+A useful special case of the pigeonhole principal with `θ = (mk α).succ`.
```

Comment:
```suggestion
A useful special case of the pigeonhole principle with `θ = (mk α).succ`.
```

Reply:
Not having read through this file before, I'm not sure what `θ` refers to here: it doesn't appear in `infinite_pigeonhole`.

Reply:
Removed from docstring.